sensing/pointcloud_preprocessor/include/pointcloud_preprocessor/distortion_corrector/distortion_corrector.hpp

```diff
@@ -48,6 +48,8 @@ namespace pointcloud_preprocessor
 using rcl_interfaces::msg::SetParametersResult;
 using sensor_msgs::msg::PointCloud2;
 
+static const int SINCOS_TABLE_SIZE = 36000;
```

Comment:
[Q] How is the resolution of 0.01 [deg] determined?  (Is it based on the resolution of the LiDAR currently in use?)

Reply:
It's an arbitrary value. 
Should it be parameterized so that it can be set for each type of LiDAR, for example?
Or if you prefer a constant value, could you tell me how to determine the appropriate setting values in terms of the quality of logical outputs?

Reply:
Sorry for my late reply.
I think constant value is fine. Since the minimum angular resolution for our LiDAR is 0.1 deg, the 0.01 deg resolution in software is reasonable.